fh-messaging/lib/fhsrv.js

```diff
@@ -175,7 +178,6 @@ var msg = (function() {
     });
   });
 
-
   router.post('/:topic', function(req, res) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Function has a complexity of 21 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AFunctionComplexity)